repo_health/check_dependabot_alerts.py

```diff
@@ -0,0 +1,78 @@
+"""
+Checks repository open dependabot alert and collects metrics.
+"""
+
+import json
+import logging
+import os
+
+import requests
+from pytest_repo_health import add_key_to_metadata
+
+from .utils import github_org_repo
+
+logger = logging.getLogger(__name__)
+
+MODULE_DICT_KEY = "dependabot_alerts"
+
+def get_github_dependabot_api_response(org_name, repo_name):
+    """
+    get the repos dependabot information using api
+    """
+
+    response = requests.get(
+        url=f'https://api.github.com/repos/{org_name}/{repo_name}/dependabot/alerts',
+        headers={'Authorization': f'Bearer {os.environ["GITHUB_TOKEN"]}'}
+    )
+
+    if response.status_code != 200:
+        logger.error(
+                "An error occurred while fetching %s. status code %s content info %s.",
+                repo_name,
+                response.status_code,
+                response.content
+        )
+        return None
+
+    return response.json()
+
+@add_key_to_metadata((MODULE_DICT_KEY, ))
+def check_dependabot_alert_stats(all_results, git_origin_url):
+    """
+    Checks repo stats for dependabot alerts
+    """
+    org_name, repo_name = github_org_repo(git_origin_url)
+    stats = compile_dependabot_stats(org_name, repo_name)
+
+    for stat_key, stat in stats.items():
+        all_results[MODULE_DICT_KEY][stat_key] = stat
+
```

Comment:
I think this would work? Haven't tested it.
```suggestion
    all_results[MODULE_DICT_KEY]['dependabot_alerts'] = stats
```